internal/pkg/model/variables.go

```diff
@@ -18,7 +18,12 @@ type VariablesFromRelation struct {
 }
 
 // VariablesValuesForRelation - variables default values for target configuration.
-type VariablesValuesForRelation struct{}
+type VariablesValuesForRelation struct {
+	// Prevent empty struct,
+	// because two distinct zero-size variables may have the same address in memory.
+	// And it fails with deepcopy, because pointers are same.
+	X bool
```

Comment:
Tak toto bola dobra past :) 

Riesil som vcera dost dlho, ze mi nefunguje klonovanie struktury `VariablesValuesForRelation`.
Kod vyzeral byt OK.

Struktura je v `memory` nejaka oblast, v ktorej su `fields` znamej dlzky ulozene za sebou.
Problem nastava, ked je struktura prazdna, nema ziadne fields.
Potom nema ani ziadnu oblast v `memory`.

Go to riesi tak, ze pre kazdy typ struktury ma jednu adresu pre vsetky instancie.
Ak je struktura prazdna. A tak mi pada klonovanie.
Preto som tu pridal tento field, ktory nema ziaden vyznam.

Vyvetlene tu: https://stackoverflow.com/a/48052903

Skus: https://go.dev/play/
![image](https://user-images.githubusercontent.com/19371734/151511864-d47df989-6ead-498b-9841-740cb405e13e.png)